apps/admin/backend/src/server.ts

```diff
@@ -68,20 +68,28 @@ function constructDippedSmartCardAuthMachineState(
 
 function buildApi(auth: DippedSmartCardAuthApi, workspace: Workspace) {
   return grout.createApi({
-    getAuthStatus: () =>
-      auth.getAuthStatus(constructDippedSmartCardAuthMachineState(workspace)),
+    getAuthStatus() {
+      return auth.getAuthStatus(
+        constructDippedSmartCardAuthMachineState(workspace)
+      );
+    },
```

Comment:
Noticed that all Grout methods before I started adding mine for auth use function syntax instead of fat arrow syntax so decided to be consistent

Reply:
Oh nice catch